mne/time_frequency/spectrum.py

```diff
@@ -1449,12 +1544,19 @@ class EpochsSpectrumArray(EpochsSpectrum):
 
     Parameters
     ----------
-    data : array, shape (n_epochs, n_channels, n_freqs)
-        The power spectral density for each channel in each epoch.
+    data : ndarray, shape (n_epochs, n_channels, [n_tapers], n_freqs, [n_segments])
+        The spectra for each channel in each epoch.
     %(info_not_none)s
     %(freqs_tfr_array)s
     %(events_epochs)s
     %(event_id)s
+    method : str
+        The spectral estimation method. Default ``'unknown'``. Must be provided if data
+        contains unaggregated tapers (``method='multitaper'``) or segments
+        (``method='welch'``).
```

Comment:
see above comments re: not passing `method` to the SpectrumArray constructors (and passing `dims` instead).  WDYT about that approach?

Reply:
Yeah, I like it. Have changed this.